### ISSUE TITLE
Delete video file from server without removing from index

### ISSUE THREAD

**Issue by ctlaltdefeat** [Chocobozzz/PeerTube#601]
Hello,

If I understand correctly, if a video gains enough traction to the point where it is constantly being seeded by viewers/other parties, then the video being seeded by the server using WebSeed is not strictly necessary. In this case, it may be ideal to be able to remove the static content from the instance while keeping the video in the DB. I imagine that if such a video does not currently have connectable peers then some message could be displayed to the user explaining this in simpler terms. 

Apologies if something like this is already implemented, I have not dug deep. If not - consider this an enhancement proposal. It increases the number of videos a single instance can support, among other things.

**Comment by rigelk** [comment]
Hi @ctlaltdefeat - deleting a video from the originating instance is out of focus until we have a redundancy mechanism (1).

By (1), I mean that right now there are very few BitTorrent clients that support WebTorrent, for long term seeding. Likewise, they are not integrated with PeerTube and you still need to download the torrent manually so it's unlikely a lot of people are using it to the point your feature proposal could be interesting. Right now the only seeding that we have integrated fully is short term seeding while someone is watching the video in the browser. But short term means that it's not a reliable to qualify as a redundancy mechanism/seed for later viewers.

There are plenty of ways we can help long term seeding emerge, but all require "some" work first :)
